python/ray/scripts/scripts.py

```diff
@@ -664,6 +683,19 @@ def get_head_ip(cluster_config_file, cluster_name):
     click.echo(get_head_node_ip(cluster_config_file, cluster_name))
 
 
+@cli.command()
+@click.argument("cluster_config_file", required=True, type=str)
+@click.option(
+    "--cluster-name",
+    "-n",
+    required=False,
+    type=str,
+    help="Override the configured cluster name.")
+def get_worker_ips(cluster_config_file, cluster_name):
+    click.echo('\n'.join(
+        get_worker_node_ips(cluster_config_file, cluster_name)))
```

Comment:
nit: splitting this into two lines instead of inlining can help with debuggability